src/ploomber/util/dotted_path.py

```diff
@@ -476,7 +476,10 @@ def from_spec(cls, dotted_path_spec, strict=False):
             )
 
     def get_kwargs(self):
-        return self.dict(exclude={"dotted_path"})
+        if int(pydantic.__version__.split(".")[0]) >= 2:
```

Comment:
add an entry to the changelog to specify that we now support pydantic 2.0

Reply:
Added Changelog